quantum/eeconfig.c

```diff
@@ -51,7 +51,11 @@ void eeconfig_init_quantum(void) {
 
     eeprom_update_word(EECONFIG_MAGIC, EECONFIG_MAGIC_NUMBER);
     eeprom_update_byte(EECONFIG_DEBUG, 0);
+#if defined(DEFAULT_LAYER_BITMASK_ENABLE)
     default_layer_state = (layer_state_t)1 << 0;
+#else
+    default_layer_state = 0;
+#endif
     eeprom_update_byte(EECONFIG_DEFAULT_LAYER, default_layer_state);
```

Comment:
Setting the global `default_layer_state` to zero would be incorrect as its expected to be a bitmask of enabled layers, and should probably look more like...
```suggestion
    default_layer_state = (layer_state_t)1 << 0;
#if defined(DEFAULT_LAYER_BITMASK_ENABLE)
    eeprom_update_byte(EECONFIG_DEFAULT_LAYER, default_layer_state);
#else
    eeprom_update_byte(EECONFIG_DEFAULT_LAYER, 0);
#endif
```